test/ParserTest.php

```diff
@@ -41,6 +41,16 @@ public function parserProvider()
 
         $eqAST = new AST\EqOperation;
         $eqAST->setProperty('name');
+        $eqAST->setValue('"12"');
+        $tests['simple eq with numeric in double quotes'] = array('eq(name,"12")', $eqAST);
```

Comment:
Will add both to the testcases .. will see, if they break s.th. thx for the hint.